automation/jleechanorg_pr_automation/openai_automation/codex_github_mentions.py

```
@@ -331,33 +411,96 @@ async def find_github_mention_tasks(self) -> List[Dict[str, str]]:
+            f'button:has-text("{label}")',
+            f'a:has-text("{label}")',
+            f'[role="tab"]:has-text("{label}")',
+        ]
+        for selector in selectors:
+            locator = self.page.locator(selector)
+            try:
+                if await locator.count() > 0:
+                    await locator.first.click()
+                    await asyncio.sleep(2)
+                    return True
+            except Exception as exc:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)